Resources/Learn Bayes/qml/qml_components/LStestinginference.qml

```diff
@@ -220,8 +250,8 @@ Section
 								 plotsPriorMarginalType.currentText == "HPD"
 						enabled: plotsPriorMarginalCI.checked
 						name: "plotsPriorMarginalCoverage"
-						label: qsTr("probability")
-						fieldWidth: 40
+						label: qsTr("coverage")
```

Comment:
Shouldn't "coverage" be with capitalization? 
(Im just guessing and looking at other labels here though)

Reply:
Coverage has a particular frequentist meaning, see https://en.wikipedia.org/wiki/Coverage_probability

You might want to change this to credibility.

Reply:
I think that it should be fine here, it's a continuation of the "CI" checkbox menu (it would be on the same line ideally, but there wasn't enough space):
![image](https://user-images.githubusercontent.com/38475991/91044277-4edc8d00-e615-11ea-8873-6ea9b889670c.png)